docs/content.zh/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -404,6 +412,63 @@ spec:
             path: log4j-console.properties
 ```
 
+`jobmanager-session-deployment-ha.yaml`
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flink-jobmanager
+spec:
+  replicas: 1 # Set the value to greater than 1 to start standby JobManagers
+  selector:
+    matchLabels:
+      app: flink
+      component: jobmanager
+  template:
+    metadata:
+      labels:
+        app: flink
+        component: jobmanager
+    spec:
+      containers:
+      - name: jobmanager
+        image: apache/flink:{{< stable >}}{{< version >}}-scala{{< scala_version >}}{{< /stable >}}{{< unstable >}}latest{{< /unstable >}}
+        env:
+        - name: POD_IP
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: status.podIP
+        args: ["jobmanager", "$(POD_IP)"]
```

Comment:
Maybe add a comment that this will set the `POD_IP` as `jobmanager.rpc.address`.

Reply:
Or better, that this overwrites the value configured in the configuration config map.